src/axolotl/loaders/patch_manager.py

```
@@ -43,6 +44,29 @@
+def _warn_irreversible_weight_transforms(key: str, transforms: list) -> None:
+    """``save_pretrained(save_original_format=True)`` reverses registered
+    conversions; surface unreversible entries at registration instead of at
+    save time."""
+    for transform in transforms:
+        problems = []
+        if getattr(transform, "quantization_operation", None) is not None:
+            problems.append("a quantization operation")
+        for operation in getattr(transform, "operations", None) or ():
+            try:
+                _ = operation.reverse_op
+            except Exception:  # pylint: disable=broad-exception-caught
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)